src/tests/datetime.rs

```diff
@@ -0,0 +1,21 @@
+use std::str::FromStr;
+
+#[test]
+fn rfc3339_to_datetime() {
```

Comment:
We'll want to acquire the test lock in both of these tests:
`let _guard = LOCK.run_concurrently();`